src/io/output_time.impl.hh

```diff
@@ -166,6 +166,45 @@ public:
             }
 
     /**
+     * Output data element on given index @p idx. Method for writing data
+     * to output stream in binary format.
+     *
+     * \note This method is used only by MSH file format.
+     */
+    void print_binary(ostream &out_stream, unsigned int idx) override
+    {
+    	ASSERT_LT(idx, this->n_values).error();
+    	out_stream << std::fixed << std::setprecision(10);
+        ElemType *ptr_begin = this->data_ + n_elem_ * idx;
+        for(ElemType *ptr = ptr_begin; ptr < ptr_begin + n_elem_; ptr++ ) {
+            char* b = reinterpret_cast<char*>(&(*ptr));
+            for(unsigned int i=0; i<sizeof(ElemType); i++) {
+            	out_stream << b[sizeof(ElemType)-1-i];
```

Comment:
- binary format use also appended VTK
- print_binary and print_binary_all methods assume stream open as binary
- output of appended data is done into a temporary stringstream opened as binary, and then written into the output stream as string.